src/utils/database.ts

```diff
@@ -1,12 +1,10 @@
 import { QuickDB, MySQLDriver } from "quick.db";
-import dotenv from "dotenv";
-dotenv.config();
 
-async function database() {
+export async function database() {
   const mysql = new MySQLDriver({
     host: process.env.MYSQL_HOST,
     user: process.env.MYSQL_USERNAME,
-    port: Number(process.env.MYSQL_PORT),
+    port: parseInt(process.env.MYSQL_PORT),
```

Comment:
Good 👍  always better to use parseInt (hmm totally not my suggestion hmmmmmm 🤫 )